demos/prefsServerIntegration/css/style.css

```diff
@@ -0,0 +1,4 @@
+/* In the preferences server integration demo, show the selfVoicingToggle button on the congratulations page */
+.gpii-fd-lastPanel.gpii-fd-prefsServerIntegration .gpii-fd-selfVoicingToggle {
```

Comment:
I'm not sure if this is specific to the integration or if it should be shown on the congratulation page for all of the demos.